linux_pipeline/Jenkinsfile_azure_lis_hotfix_performance

```diff
@@ -1,15 +1,10 @@
 #!/usr/bin/env groovy
-def PerfTestList = [
-    ["TCPNetworkPerformance_Multiclient", "PERF-NETWORK-TCP-THROUGHPUT-MULTICLIENTS-NTTTCP-Synthetic,PERF-NETWORK-TCP-THROUGHPUT-MULTICLIENTS-NTTTCP-SRIOV","westus2"],
-    ["TCPNetworkPerformance_SingleClient","PERF-NETWORK-TCP-LATENCY-Synthetic,PERF-NETWORK-TCP-LATENCY-SRIOV,PERF-NETWORK-TCP-SINGLE-CONNECTION-THROUGHPUT-SYNTHETIC,PERF-NETWORK-TCP-SINGLE-CONNECTION-THROUGHPUT-SRIOV",
-    "westus2"],
-    ["Storage4K", "PERF-STORAGE-4K-IO", "westus2"],
-    ["Storage1024K", "PERF-STORAGE-1024K-IO", "northeurope"],
-    ["NVME4K", "PERF-NVME-4K-IO", "westus2"],
-    ["UDPNetworkPerformance_Multiclient", "PERF-NETWORK-UDP-1K-THROUGHPUT-MULTICONNECTION-NTTTCP-Synthetic,PERF-NETWORK-UDP-1K-THROUGHPUT-MULTICONNECTION-NTTTCP-SRIOV", "westus2"],
-    ["PERF-SYSCALL-BENCHMARK", "PERF-SYSCALL-BENCHMARK", "westus2"]
-]
-def PerfTestCount = PerfTestList.size()
+
+def TestLocation = TEST_LOCATION.replaceAll("[\n\r]", "");
```

Comment:
We are taking TEST_LOCATION value as single string. No need to replaceAll()

Reply:
Done